core/src/main/java/org/pentaho/di/core/database/DatabaseMeta.java

```diff
@@ -3142,4 +3142,8 @@ public String getNamedCluster() {
   public void setNamedCluster( String namedCluster ) {
     databaseInterface.setNamedCluster( namedCluster );
   }
+
+  public Object makeClone() {
+    return clone();
+  }
```

Comment:
This doesn't override or implement some interface? 
If this is an override, it needs `@Override`